docs/source/guides/images_explorer_gan.md

```diff
@@ -0,0 +1,110 @@
+## Track and compare GANs with Aim
```

Comment:
Maybe make it 
End-to-end images tracking and explore with GANs ?

Reply:
too long, breaks inside the sidebar